VERSION

```diff
@@ -0,0 +1 @@
+1.9.1
```

Comment:
This should be the next version, 1.9.2

https://github.com/chef/knife-ec2/pull/525#issuecomment-392798723